### PR TITLE
To-Do-List-Setup With Webpack & ES6

### PR DESCRIPTION
**Pull request by Tobinchilongo**
## In this `PR` I am  implementing the following functionalities...

- Adding a new item.
- Removing a selected item.
- Marking a selected item as complete.
- Removing all items marked as complete at once.
- Reordering a selected item (as drag-and-drop) [as an additional feature].
- Use webpack to bundle JavaScript.
- Using of proper ES6 syntax.
- Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  const element = document.createElement('div');
+
+  // Lodash, now imported by this script
+  element.innerHTML = _.join([], ' ');
+  element.classList.add('hello');
+
+  return element;
+}
+
+document.body.appendChild(component());
```
Comment (D4nielJ) [inline src/index.js]: - [x] Please implement an array of some simple to do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number].
- [x] Please implement a function that iterates over the tasks array and populates HTML list item element for each task.
- [x] On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.

> You must have an empty To DO List place holder on your index.html file to achieve this.

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @Tobinchilongo, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: :rocket: 
- Styling looks great
- Webpack is set up correctly! :sparkles:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Tobinchilongo) [reply src/index.js]: ### Implemented all the required changes.

Thank you very much for the positive feedback.

**Review by julie-ify — APPROVED** [review]
## Hi @Tobinchilongo, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**